enhancements/ingress/admin-configured-ca-bundle.md

```diff
@@ -0,0 +1,314 @@
+---
+title: admin-configured-ca-bundle
+authors:
+  - "@bhb"
+reviewers:
+  - "@candita" ## reviewer for cluster-ingress-operator component
+  - "@Miciah" ## reviewer for cluster-ingress-operator component
+  - "@tgeer" ## reviewer for CFE
+approvers:
+  - "@candita" ## approval from cluster-ingress-operator component
+  - "@Miciah" ## approval from cluster-ingress-operator component
+  - "@jerpeter1" ## approval from CFE
+api-approvers:
+  - None
+creation-date: 2023-11-02
+last-updated: 2023-12-06
+tracking-link:
+  - https://issues.redhat.com/browse/RFE-2182
+  - https://issues.redhat.com/browse/OCPSTRAT-431
+  - https://issues.redhat.com/browse/NE-761
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Support for admin configured CA trust bundle in Ingress Operator
+
+## Summary
+
+This enhancement allows an OpenShift cluster administrator to configure OpenShift router with custom
+trusted Certificate Authorities (CAs) to use for verifying services' certificates of re-encrypt routes.
+The enhancement proposes concatenating the custom and OpenShift service CA certificates together for the
+purposes of a common CA to be used for certificate validation on re-encrypt routes.
+
+## Motivation
+
+When an OpenShift administrator creates a route with the re-encrypt termination type, they have to also
```

Comment:
doesn't a namespace editor create the route?

Reply:
Sorry, I am not sure what `namespace editor` is. But this is for the routes created by an admin to expose a deployed service.

Reply:
> Sorry, I am not sure what `namespace editor` is. But this is for the routes created by an admin to expose a deployed service.

routes are created by low-privileged users of a cluster, not the cluster-admin.  Those low privileged users include editors who are allowed to create pods, services, and routes, but lack visibility into cluster configuration.

Reply:
Sorry, I used `OpenShift administrator` in generic way as those who have access to cluster and have sufficient permissions to provision required resources and I didn't think it in line with the roles.

Reply:
Still outstanding.  We have different roles for our product and this is a low-privilege operation.  The openshift administrator is a high privilege account and who performs an action is considered in places in our API.  Please use the thing you mean.  I think you mean "namespace editor"